src/components/heading/heading.style.js

```diff
@@ -1,10 +1,30 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import Icon from '../icon';
 import StyledIcon from '../icon/icon.style';
 import baseTheme from '../../style/themes/base';
+import StyledLink from '../link/link.style';
+import { isClassic } from '../../utils/helpers/style-helper';
 
 const StyledHeading = styled.div`
   width: 100%;
+  &&&& ${StyledLink} {
```

Comment:
`&&&&` – that's quite a lot of [specificity](https://www.styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity)!

Reply:
I'm a specific kind of guy 😉, good point was trying out something will scale it back to `&&`

Reply:
I think it needs to be that specific as it doesn't override the styling as needed when the `Heading` is used in `Pages` as `margin-top: -8px;` is applied in the `.scss` file

Reply:
OK :thumbsup: